lib/ssl/src/tls_server_connection_1_3.erl

```diff
@@ -363,6 +363,8 @@ wait_eoed(Type, Msg, State) ->
                  term(), #state{}) ->
           gen_statem:state_function_result().
 %%--------------------------------------------------------------------
+connection(info, Msg, State) ->
+    tls_gen_connection:handle_info(Msg, connection, State);
```

Comment:
Same question as for client.

Reply:
To avoid those 2 function calls?

Reply:
Now I see.

Reply:
But maybe it should be gen_info that is called, lets continue this when we see how #9273 turns out.  To ensure that error handling is done correctly.

Reply:
?

Reply:
I think we need to use tls_gen_connection:gen_info instead of tls_gen_connection:handle_info so that error handling works as intended.

Reply:
Ingela have a PR which fixes this in alot of places, so it will be fixed in that branch.

Reply:
I will make a new PR that fixes it.  I made other such fixes in my other PR, but it is for maint so I can not fix it there.